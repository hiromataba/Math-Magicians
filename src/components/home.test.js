import React from 'react';
import renderer from 'react-test-renderer';
import Home from './home';

it('expect quote to match snapshot', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
