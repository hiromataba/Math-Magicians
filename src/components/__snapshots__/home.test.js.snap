// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect quote to match snapshot 1`] = `
<div
  className="home"
  style={
    Object {
      "color": "#363c48",
      "lineHeigth": "990px",
      "margin": "100px",
    }
  }
>
  <h3
    style={
      Object {
        "fontSize": "35px",
        "marginBottom": "50px",
      }
    }
  >
    Welcome to Math Magicians
  </h3>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptate libero est error? Nesciunt impedit incidunt, optio consequatur quasi, dignissimos voluptates veritatis velit illum repellat itaque laboriosam animi assumenda harum obcaecati magnam eos doloremque unde ex et sequi, maxime sed quis? Minus, distinctio at? Perferendis consectetur quasi accusantium aliquam, quaerat pariatur!
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate non atque quos debitis deserunt similique itaque porro. Culpa aut, voluptatem, assumenda incidunt quaerat eos cum sed architecto reprehenderit voluptates unde enim vero, distinctio nobis temporibus labore illo officiis dolores. Vero quod a vitae voluptates blanditiis id dolorem commodi assumenda sit!
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptate libero est error? Nesciunt impedit incidunt, optio consequatur quasi, dignissimos voluptates veritatis velit illum repellat itaque laboriosam animi assumenda harum obcaecati magnam eos doloremque unde ex et sequi, maxime sed quis? Minus, distinctio at? Perferendis consectetur quasi accusantium aliquam, quaerat pariatur!
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate non atque quos debitis deserunt similique itaque porro. Culpa aut, voluptatem, assumenda incidunt quaerat eos cum sed architecto reprehenderit voluptates unde enim vero, distinctio nobis temporibus labore illo officiis dolores. Vero quod a vitae voluptates blanditiis id dolorem commodi assumenda sit!
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptate libero est error? Nesciunt impedit incidunt, optio consequatur quasi, dignissimos voluptates veritatis velit illum repellat itaque laboriosam animi assumenda harum obcaecati magnam eos doloremque unde ex et sequi, maxime sed quis? Minus, distinctio at? Perferendis consectetur quasi accusantium aliquam, quaerat pariatur!
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate non atque quos debitis deserunt similique itaque porro. Culpa aut, voluptatem, assumenda incidunt quaerat eos cum sed architecto reprehenderit voluptates unde enim vero, distinctio nobis temporibus labore illo officiis dolores. Vero quod a vitae voluptates blanditiis id dolorem commodi assumenda sit!
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptate libero est error? Nesciunt impedit incidunt, optio consequatur quasi, dignissimos voluptates veritatis velit illum repellat itaque laboriosam animi assumenda harum obcaecati magnam eos doloremque unde ex et sequi, maxime sed quis? Minus, distinctio at? Perferendis consectetur quasi accusantium aliquam, quaerat pariatur!
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate non atque quos debitis deserunt similique itaque porro. Culpa aut, voluptatem, assumenda incidunt quaerat eos cum sed architecto reprehenderit voluptates unde enim vero, distinctio nobis temporibus labore illo officiis dolores. Vero quod a vitae voluptates blanditiis id dolorem commodi assumenda sit!
  </p>
</div>
`;
